web/src/main/react-dashboard/src/components/DeploymentPatternView.js

```diff
@@ -28,11 +28,11 @@ import {
 } from 'material-ui/Table';
 import Subheader from 'material-ui/Subheader';
 import SingleRecord from './SingleRecord.js';
-import { add_current_deployment, add_current_infra } from '../actions/testGridActions.js';
+import {add_current_deployment, add_current_infra} from '../actions/testGridActions.js';
```

Comment:
Since '../actions/testGridActions.js' is used in multiple places shall we add a constant for this?

Reply:
Hi azinneera, since it is referring to a import line of the class which happens before starting the class block, it is not possible to refer a constant from the class or from another import.